bin/node/runtime/src/lib.rs

```diff
@@ -520,9 +521,14 @@ parameter_types! {
 	pub const SignedPhase: u32 = EPOCH_DURATION_IN_BLOCKS / 4;
 	pub const UnsignedPhase: u32 = EPOCH_DURATION_IN_BLOCKS / 4;
 
-	// fallback: no need to do on-chain phragmen initially.
-	pub const Fallback: pallet_election_provider_multi_phase::FallbackStrategy =
-		pallet_election_provider_multi_phase::FallbackStrategy::Nothing;
+	// signed config
+	pub const SignedMaxSubmissions: u32 = 10;
+	pub const SignedRewardBase: Balance = 1 * DOLLARS;
+	pub const SignedDepositBase: Balance = 1 * DOLLARS;
+	pub const SignedDepositByte: Balance = 1 * CENTS;
```

Comment:
@AndreiNavoichyk 
I can't say how does this affect as. Could you please verify this changes by yourself?

Reply:
@MRamanenkau 
Could you share a PR where Parity introduced this changes?

Reply:
[Here it is](https://github.com/paritytech/substrate/pull/7910)

Reply:
[We need min 4 validators to run staking election](https://substrate.stackexchange.com/questions/630/validator-not-being-onboarded-due-to-election-nofallback-error-possibly)